src/components/Display.js

```diff
@@ -0,0 +1,10 @@
+import React from "react"
+
+function Display() {
+  return (
+    <input type='text'/>
+    
+  )
+}
+

```

Comment:
- this component should have an incoming prop called `result` [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/incomingProps.png)]
- `result` should be a string (validate this with `propTypes`) [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/validateAsString.png)]

- `result` should have default value of `"0"` (enforce this with `defaultProps`) [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/DefaultProp.png)]